docs/_static/auth/iam.yaml

```diff
@@ -562,6 +562,290 @@ paths:
                 "$ref": "#/components/schemas/VerifiableCredential"
         default:
           $ref: '../common/error_response.yaml'
+  /iam/{did}/start-oid4vci-issuance:
```

Comment:
should be at /internal/auth/v2/{did}/request-credential

Reply:
Done